types/google.accounts/index.d.ts

```diff
@@ -229,15 +286,40 @@ declare namespace google.accounts {
             prompt?: '' | 'none' | 'consent' | 'select_account';
 
             /**
+             * Optional, defaults to true. If set to false, more granular Google
+             * Account permissions would be disabled for OAuth client IDs
+             * created before 2019. If both enable_granular_consent and
+             * enable_serial_consent are set, only enable_granular_consent value
+             * would take effect and enable_serial_consent value would be
+             * ignored.
+             *
+             * No effect for newer OAuth client IDs, since more granular
+             * permissions is always enabled for them.
+             */
+            enable_granular_consent?: boolean;
+
+            /**
+             * Deprecated, you should use enable_granular_consent instead.
              * Optional, defaults to true.
```

Comment:
as above, this default may no longer be the case, and it is not mentioned in the current google documentation.